ETSN20_text-main/textSearchProject/src/textSearchProject/textSearch.java

```diff
@@ -0,0 +1,80 @@
+package textSearchProject;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.net.URL;
+
+public class textSearch {
+
+	public static void main(String[] args) {
+//		Scanner scanner = new Scanner(System.in);
+//		String inputString = "";
+		String testString = "search bee bee.txt";
+//		while (!legitInput(inputString)) {
+//			System.out.println("Enter a search query in the form: search pattern file");
+//			inputString = scanner.nextLine();
+//		}
+		String inputString = testString;
```

Comment:
testString and inputString are the same, duplicated code.

Reply:
Fixed